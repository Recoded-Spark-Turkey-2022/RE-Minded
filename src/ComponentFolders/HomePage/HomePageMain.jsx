import React from 'react';
import ContentA from './ContentA';
import ContentB from './ContentB';
import ContentC from './ContentC';
import ContentD from './ContentD';
import ContentE from './ContentE';

function HomePageMain() {
  return (
    <>
      <ContentA />
      <ContentB />
      <ContentC />
      <ContentD />
      <ContentE />
    </>
  );
}

export default HomePageMain;
