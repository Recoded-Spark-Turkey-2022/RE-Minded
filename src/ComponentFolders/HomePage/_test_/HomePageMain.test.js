import React from 'react';
import renderer from 'react-test-renderer';
import HomePageMain from '../HomePageMain'

test('Home Page snapshot test', () => {
  const component = renderer.create(<HomePageMain />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
