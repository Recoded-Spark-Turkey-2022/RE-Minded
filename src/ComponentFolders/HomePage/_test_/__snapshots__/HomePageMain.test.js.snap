// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page snapshot test 1`] = `
Array [
  <div
    style={
      Object {
        "backgroundImage": "url(Background.svg)",
        "backgroundPosition": "top",
        "backgroundRepeat": "no-repeat",
        "backgroundSize": "cover",
        "height": "100vh",
        "top": 0,
        "width": "100vw",
      }
    }
  >
    <div
      className="flex flex-col pt-40   items-center md:flex-col   lg:flex-row    lg:justify-evenly lg:items-center md:items-center lg:pt-72  md:pt-40"
    >
      <div
        className=" p-32 flex flex-col items-center   md:flex-col md:items-center  lg:items-center "
      >
        <p
          className="text-3xl md:text-4xl lg:text-5xl"
        >
           WE ARE HERE TO
        </p>
        <p
          className="text-6xl  md:text-7xl lg:text-9xl "
        >
          HELP
        </p>
        <button
          className="text-xl rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 md:text-2xl"
          type="button"
        >
          BOOK AN APPOINTMENT
        </button>
      </div>
      <div
        className=""
      >
        <img
          alt="Sofa"
          src="ContentAsofa.svg"
        />
      </div>
    </div>
  </div>,
  <div>
    <div
      className="bg-amber-200 pb-16 font-poppins"
    >
      <h1
        className="tracking-wider lg:text-5xl text-lg font-bold lg:w-7/12 lg:font-poppins lg:pt-20 pt-20 pl-16 sm:p-16 sm:font-poppins"
      >
        PROFESSIONAL, LICENSED, AND VETTED THERAPISTS THAT YOU CAN TRUST
      </h1>
      <img
        alt="therapists"
        className="pt-16 pb-16 pl-16"
        src="ContentBTherapists.svg"
      />
      <p
        className="pl-16 lg:text-2xl font-poppins text-sm lg:w-8/12 lg:pb-20 sm:pb-16"
      >
        Tap into the worlds largest network of licensed, accredited, and experienced therapists who can help you with a range of issues including depression, anxiety, relationships, trauma, grief, and more. with our therapists, you get the same professionalism and quality you would expect from an in-office therapist, but with the ability to communicate whenever and however you want.
      </p>
      <button
        className="m-12 text-md md:text-lg ml-16 lg:px-6 rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 "
        type="button"
      >
        BOOK AN APPOINTMENT
      </button>
    </div>
  </div>,
  <div
    className="font-poppins p-12 mb-16 mt-4  flex flex-col justify-evenly "
  >
    <h1
      className="sm:text-xl md:text-4xl xl:text-4xl p-16 mb-2"
    >
      WE CAN COMMUNICATE THROUGH
    </h1>
    <div
      className="flex flex-col md:flex-row items-center justify-evenly "
    >
      <div
        className="shadow-zinc-300 h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg  rounded-3xl md:m-12 p-4"
      >
        <img
          alt="Video Call"
          className="pt-4  mb-4 "
          src="ContentCVideoCall.svg"
        />
        <h1
          className="pt-4 text-center md:text-xl lg:text-3xl   "
        >
          Video Call
        </h1>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-lg lg:text-lg py-4"
        >
          For better experience therapists recommend video calls, but always remember that its a choice!
        </p>
        <h2
          className="text-2xl md:text-lg lg:text-2xl"
        />
        <h2
          className="text-2xl flex justify-center p-6 md:text-lg lg:text-2xl"
        />
      </div>
      <div
        className="shadow-zinc-300 h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg  rounded-3xl md:m-12 p-4"
      >
        <img
          alt="Chat"
          className="pt-4  mb-4 "
          src="ContentCChat.svg"
        />
        <h1
          className="pt-4 text-center md:text-xl lg:text-3xl   "
        >
          Chat
        </h1>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-lg lg:text-lg py-4"
        >
          Need to talk to someone? Come have a chat with your therapist!
        </p>
        <h2
          className="text-2xl md:text-lg lg:text-2xl"
        />
        <h2
          className="text-2xl flex justify-center p-6 md:text-lg lg:text-2xl"
        />
      </div>
      <div
        className="shadow-zinc-300 h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg  rounded-3xl md:m-12 p-4"
      >
        <img
          alt="Voice Call"
          className="pt-4  mb-4 "
          src="ContentCVoiceCall.svg"
        />
        <h1
          className="pt-4 text-center md:text-xl lg:text-3xl   "
        >
          Voice Call
        </h1>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-lg lg:text-lg py-4"
        >
          Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!
        </p>
        <h2
          className="text-2xl md:text-lg lg:text-2xl"
        />
        <h2
          className="text-2xl flex justify-center p-6 md:text-lg lg:text-2xl"
        />
      </div>
    </div>
  </div>,
  <div
    className="bg-cyan-50 pb-16 w-full h-max "
  >
    <h1
      className="font-bold lg:text-start p-24 pl-32 font-poppins text-3xl md:text-4xl  "
    >
      RECENT BLOGS
    </h1>
    <div
      className="flex flex-col p-0 lg:flex-row md:flex-col items-center justify-around w-full lg:h-1/3 pb-24 "
    >
      <img
        alt="LeftArrow"
        className="p-3 cursor-pointer rotate-90 lg:rotate-0 "
        src="LeftArrow.svg"
      />
      <img
        alt="Blog1"
        className="p-1 cursor-pointer  "
        src="ContentDBlog1.svg"
      />
      <img
        alt="Blog2"
        className="p-1 cursor-pointer"
        src="ContentDBlog2.svg"
      />
      <img
        alt="Blog1"
        className="cursor-pointer"
        src="ContentDBlog1.svg"
      />
      <img
        alt="RightArrow"
        className="p-3 cursor-pointer rotate-90 lg:rotate-0 "
        src="RightArrow.svg"
      />
    </div>
  </div>,
  <div
    className="font-poppins p-12 flex flex-col justify-evenly lg:h-section py-8"
  >
    <h1
      className="pl-16 font-poppins text-3xl md:text-4xl pb-8 pt-12"
    >
      PURCHASE TICKETS
    </h1>
    <h4
      className="pl-16 font-poppins text-xl mb-8 "
    >
      PURCHASE TICKETS THAT CAN BE USED TO BOOK APPOINTMENTS!
    </h4>
    <div
      className=" flex flex-col md:flex-row items-center justify-around w-full lg:h-1/3"
    >
      <div
        className="shadow-zinc-300 h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg  rounded-3xl md:m-12 p-4"
      >
        <h2
          className="text-2xl md:text-lg lg:text-2xl"
        >
          5 TICKETS
        </h2>
        <h2
          className="text-2xl flex justify-center p-6 md:text-lg lg:text-2xl"
        >
          10$
        </h2>
        <button
          className="font-poppins rounded-md box-border p-2 mb-4 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500"
          type="button"
        >
          PURCHASE
        </button>
      </div>
      <div
        className="shadow-zinc-300 h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg  rounded-3xl md:m-12 p-4"
      >
        <h2
          className="text-2xl md:text-lg lg:text-2xl"
        >
          25 TICKETS
        </h2>
        <h2
          className="text-2xl flex justify-center p-6 md:text-lg lg:text-2xl"
        >
          40$
        </h2>
        <button
          className="font-poppins rounded-md box-border p-2 mb-4 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500"
          type="button"
        >
          PURCHASE
        </button>
      </div>
      <div
        className="shadow-zinc-300 h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg  rounded-3xl md:m-12 p-4"
      >
        <h2
          className="text-2xl md:text-lg lg:text-2xl"
        >
          50 TICKETS
        </h2>
        <h2
          className="text-2xl flex justify-center p-6 md:text-lg lg:text-2xl"
        >
          70$
        </h2>
        <button
          className="font-poppins rounded-md box-border p-2 mb-4 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500"
          type="button"
        >
          PURCHASE
        </button>
      </div>
    </div>
    <div
      className=" shadow-zinc-300 font-poppins flex flex-col justify-around items-center p-4 rounded-lg h-1/3 lg:mx-16 my-4  shadow-lg "
    >
      <h1
        className="text-2xl lg:text-3xl uppercase text-center py-4"
      >
        ARE YOU A COUNSELOR?
      </h1>
      <p
        className="font-poppins text-xl w-2/4  md:text-md  text-justify md:text-center text-opacity-50"
      >
        Interested in joining our mental health platform? You decide your schedule and how much you want to work, well take care of the client referrals and billing details!
      </p>
      <button
        className="font-poppins text-center my-8 rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500"
        type="button"
      >
        LEARN MORE
      </button>
    </div>
  </div>,
]
`;
